api/releases.go

```diff
@@ -288,6 +290,17 @@ func releaseDeployHandler(c echo.Context) error {
 		err = deployRelease(release, releaseRollout)
 		if err != nil {
 			log.WithError(err).Error("failed release rollout")
+		} else {
+			slack.PostLogMessage(fmt.Sprintf("Running regressions for *%s* now", environment.Name), log.InfoLevel)
+			buildParameters := make(map[string]string)
+			if environment.Branch == "master" {
```

Comment:
The branch specific logic will not work for an OS project as not everyone uses the same branching strategy.  We need to make it more generic through app configuration.